module-types/Datacom/PSU-600-AC-F.yaml

```diff
@@ -0,0 +1,10 @@
+---
+manufacturer: Datacom
+model: PSU 600 AC-F
+part_number: 820.0018.xx
+comments: AC power supply with 100-240VAC and 50/60Hz input and the ability to provide 600W of power. Features forced ventilation with airflow exiting from
+  the power supply. Supports hot-swap and operation in redundant mode.
+power-ports:
+  - name: PSU
```

Comment:
Please update this name to use the `{module}` definition

Reply:
sorry, but I don't understand how it should be